daily-challenges/2024-10/21/nhat.spec.ts

```
@@ -0,0 +1,128 @@
+import { test as base, expect, Locator, Page } from "@playwright/test";
+
+class Utils {
+  static getNumberFromString (str: string): number {
+    return Number(str.replace(/\D/g, ''));
+  };
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)